packages/configuration-validation/src/lib.js

```diff
@@ -49,14 +50,16 @@ configUtil.buildConfigObject = (config) => {
   let tokenManager = config.tokenManager;
   const autoRenew = config.autoRenew || config.auto_renew;
   const storage = config.storage;
-  if (storage || autoRenew) {
+  const disableHttpsCheck = config.testing ? config.testing.disableHttpsCheck : false;
+  if (storage || autoRenew || disableHttpsCheck) {
     // Properties already defined within the "tokenManager" section will not be overwritten
```

Comment:
If any one of them is defined,  ALL of them will not be set?  Is that intended

Reply:
This logic is for creating a `tokenManager` section if it does not exist. If none of these are defined, we will not create an empty object.